src/systemtest/resources/sinceBeginningDateRange/expected/reposense_testrepo-Delta_master/commits.json

```diff
@@ -781,13 +1279,29 @@
             "hash": "741c2631fca2f084d0d083d5607e985483a69381",
             "messageTitle": "DevGuide: Updated sort, toggle privacy and lock MTM",
             "messageBody": "",
+            "fileTypesAndContributionMap": {
+              "adoc": {
+                "insertions": 253,
+                "deletions": 23
+              }
+            },
             "insertions": 253,
             "deletions": 23
           },
           {
             "hash": "547828b327324784581897b933e477227a972789",
             "messageTitle": "Collate, PPP",
             "messageBody": "",
+            "fileTypesAndContributionMap": {
+              "java": {
+                "insertions": 1,
+                "deletions": 0
+              },
+              "adoc": {
+                "insertions": 165,
+                "deletions": 62
+              }
+            },
             "insertions": 166,
```

Comment:
are the total value still in used? if not can consider removing them

Reply:
Yes, we can remove them since we can sum up all filetype contributions. Will do so.

Reply:
total insertions and deletions value of all commitResults are removed. Added some logic in the front end to get those data initially instead.

Reply:
@eugenepeh Note that the insertions and deletions of each commit (each commit contains multiple commitResults) are still used in the commits.json files. Should we also remove them and populate the numbers in `api.js` during initialization?

Reply:
Update: can't really remove the insertions and deletions of each **commit** as they are required in the backend to calculate the variance.